sample/java-spring-boot-mvc/build.gradle

```diff
@@ -1,25 +1,41 @@
 buildscript {
     ext {
-        springBootVersion = '1.5.4.RELEASE'
+        springBootVersion = '2.1.3.RELEASE'
     }
     repositories {
-        mavenLocal()
+        // mavenLocal()
         mavenCentral()
         jcenter()
     }
     dependencies {
         classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
         classpath 'com.benjaminsproule:swagger-gradle-plugin:+'
+
+        // additional openapi stuff
+        classpath "org.openapitools:openapi-generator-gradle-plugin:4.0.0-beta2"
```

Comment:
It would be nice if I could get the plugin to do all this without another plugin, but for now a new sample would be better